homework16-inheritance.py

```diff
@@ -0,0 +1,221 @@
+# Task 1
+class Person:
+    def __init__(self,first_name,last_name,age,high,gender,eyes,hair_color):
+        self.first_name = first_name
+        self.last_name= last_name
+        self.age= age
+        self.high= high
+        self.gender= gender
+        self.eyes= eyes
+        self.hair_color=hair_color
+class Student:
+        def __init__(self,class_number):
+            self.class_number=class_number
+class Teacher:
```

Comment:
Why doesn't the `Teacher` class inherit the `Person` and extend it? Aren't teachers people?
https://images3.memedroid.com/images/UPLOADED499/5e34af8a92f43.jpeg
Okay, jokes aside, please inherit the `Student` and `Teacher` from the `Person` and extend the `Person`'s `__init__`. Important thing here: use `*args` and `*kwargs` here for the extension, call `Person`'s `__init__` in `__init__`s of the `Student` and `Teacher` in a way that allows you to modify the `Person`'s init without the need to modify the `__init__`s of the child classes.